deploy/examples/import-external-cluster.sh

```diff
@@ -93,6 +95,15 @@ function checkEnvVars() {
   fi
 }
 
+function importClusterID() {
+  if [ -n "$RADOS_NAMESPACE_CLUSTER_ID" ]; then
+    CLUSTER_ID_RBD=$(kubectl -n "$NAMESPACE" get cephblockpoolradosnamespace.ceph.rook.io/"$RADOS_NAMESPACE_CLUSTER_ID" -o jsonpath='{.status.info.clusterID}')
```

Comment:
Can't we just use $NAMESPACE for the cluster ID instead of querying it from the CR status? We already have to assume the $NAMESPACE in this command in the first place, so won't $NAMESPACE always be the same as these rbd and cephfs cluster IDs that are being extracted?

Reply:
It also appears these are set up on lines 25-26 to $NAMESPACE, so would this import ever be triggered?

Reply:
If the `NAMESPACE` is used a normal cephfs or rbd storage class would be created,

But if the user has used `rados-namespace` or `subvolume group` flag then the storage class of subvolumes or rados namespace would be created, to create them we need the  `cluster ID` from their status to add to the storage class cluster ID.

`importClusterID` function is called in the bottom where we call other functions